.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,67 @@
+name: Node.js integration
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  integration:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, macos-14, ubuntu-latest]
+        python: ["3.8", "3.10", "3.12"]
+        exclude:
+          - os: macos-14
+            python: "3.8"
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Clone gyp-next
+        uses: actions/checkout@v4
+        with:
+          path: gyp-next
+      - name: Clone nodejs/node
+        uses: actions/checkout@v4
+        with:
+          repository: nodejs/node
+          path: node
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+          allow-prereleases: true
+      - name: Replace gyp in node
+        shell: bash
+        run: |
+          rm -rf node/tools/gyp
+          cp -r gyp-next node/tools/gyp
+      - name: Run configure
+        shell: bash
+        run: |
+          cd node
+          ./configure
+
+  integration-windows:
```

Comment:
These Windows jobs take a long time!

Reply:
I think it could be enough to run the configure step instead of running all the build. In this way it could be much faster.

Reply:
Updated to run `configure` only.